src/shared-components/Card/ListItemsSwitch/ListItemsSwitch.tsx

```diff
@@ -0,0 +1,26 @@
+import {
+  List,
+  ListItem,
+  ListItemSecondaryAction,
+  ListItemText,
+  Switch,
+} from '@mui/material';
+
+export const ListItemsSwitch = ({
+  title,
+  description,
+}: {
+  title: string;
+  description: string;
+}) => {
+  return (
+    <List>
+      <ListItem>
+        <ListItemText primary={title} secondary={description}></ListItemText>
+        <ListItemSecondaryAction>
+          <Switch />
```

Comment:
To define in switch is toggled on or off, you'd need to pass a function to this component as a prop. Whenever component is used, the parent should support logic for using the state of the switch.

Reply:
Done